docs/getting-started.rst

```diff
@@ -285,19 +285,19 @@ Let's start by bringing up a Prometheus instance ourselves, to scrape our applic
 
 .. code-block:: yaml
 
-    # prometheus.yml
+    # /tmp/prometheus.yml
```

Comment:
FWIW I think it's better if we just assume everything is in (or relative to) pwd, and not all systems have a `/tmp/`.

Reply:
The problem with that approach is that docker doesn't allow to mount a relative path a as a volume: https://github.com/moby/moby/issues/4830.